CONTRIBUTING.md

```diff
@@ -51,3 +51,79 @@ Excellent! There are a few steps you'll need to follow to get ready to submit ch
 1. If necessary, update your pull request by adding additional commits to your branch in response to
    reviewer comments and automated tests. Once the reviewers have approved your changes, they will merge
    them into the official repository.
+   
+## Terms of Use
+
+By submitting Contributions (as defined below) to this project, you agree that
+You (as defined below) and your Contributions are bound to the following terms.
+
+<p align="center">Open Reaction Database Project License Agreement</p> 
+
+In order to clarify the intellectual property license granted with Contributions
+to the Open Access Reaction Database from any person or entity, the Open
+Reaction Database Project ("ORDP") must indicate agreement to the license terms
+below. 
+
+This version of the Agreement allows an entity (the "Corporation") to
+submit Contributions to ORDP, to authorize Contributions submitted by its
+designated employees to ORDP, and to grant copyright and patent licenses
```

Comment:
Does this mean that when somebody submits, they cannot do it on behalf of themselves but rather on behalf of their employer?

Reply:
PTAL